webui/react/src/pages/FlatRuns/columns.ts

```diff
@@ -0,0 +1,504 @@
+              delimiter: ' ',
+              largest: 2,
+              serialComma: false,
+              unitMeasures: { ...DURATION_UNIT_MEASURES, ms: 1000 },
+            },
+          )
+        : '',
+      data: { kind: TEXT_CELL },
+      kind: GridCellKind.Custom,
+    }),
+    title: 'Duration',
+    tooltip: () => undefined,
+    width: columnWidths.duration,
+  },
+  externalExperimentId: {
+    id: 'externalExperimentId',
+    renderer: (record: FlatRun) => ({
+      allowOverlay: false,
+      copyData: record.experiment?.externalExperimentId ?? '',
+      data: { kind: TEXT_CELL },
+      kind: GridCellKind.Custom,
+    }),
+    title: 'External Experiment ID',
+    tooltip: () => undefined,
+    width: columnWidths.externalExperimentId,
+  },
+  externalTrialId: {
+    id: 'externalTrialId',
+    renderer: (record: FlatRun) => ({
+      allowOverlay: false,
+      copyData: String(record.externalRunId ?? ''),
+      data: { kind: TEXT_CELL },
+      kind: GridCellKind.Custom,
+    }),
+    title: 'External Trial ID',
+    tooltip: () => undefined,
+    width: columnWidths.externalTrialId,
+  },
+  forkedFrom: {
+    id: 'forkedFrom',
+    renderer: (record: FlatRun) => ({
+      allowOverlay: false,
+      copyData: String(record.experiment?.forkedFrom ?? ''),
+      cursor: record.experiment?.forkedFrom ? 'pointer' : undefined,
+      data: {
+        kind: LINK_CELL,
+        link:
+          record.experiment?.forkedFrom !== undefined
+            ? {
+                href: record.experiment?.forkedFrom
+                  ? paths.experimentDetails(record.experiment?.forkedFrom)
+                  : undefined,
+                title: String(record.experiment?.forkedFrom ?? ''),
+              }
+            : undefined,
+        navigateOn: 'click',
+        onClick: (e: CellClickedEventArgs) => {
+          if (record.experiment?.forkedFrom) {
+            handlePath(e as unknown as AnyMouseEvent, {
+              path: String(record.experiment.forkedFrom),
+            });
+          }
+        },
+        underlineOffset: 6,
+      },
+      kind: GridCellKind.Custom,
+      readonly: true,
+    }),
+    title: 'Forked From',
+    tooltip: () => undefined,
+    width: columnWidths.forkedFrom,
+  },
+  id: {
+    id: 'id',
+    renderer: (record: FlatRun) => ({
+      allowOverlay: false,
+      copyData: String(record.id),
+      cursor: 'pointer',
+      data: {
+        kind: LINK_CELL,
+        link: record.experiment?.id
+          ? {
+              href: paths.experimentDetails(record.experiment.id),
+              title: String(record.id),
+              unmanaged: record.experiment.unmanaged,
+            }
+          : undefined,
+        navigateOn: 'click',
+        onClick: (e: CellClickedEventArgs) => {
+          if (record.experiment) {
+            handlePath(e as unknown as AnyMouseEvent, {
+              path: paths.experimentDetails(record.experiment.id),
+            });
+          }
```

Comment:
I think we'd want to link to the trial (not experiment) if it's a trial of a multi-trial experiment?